internal/pkg/deploy/pipeline_test.go

```diff
@@ -241,6 +241,18 @@ func TestPipelineBuild_Init(t *testing.T) {
 				BuildspecPath:   "some/path",
 			},
 		},
+		"additional policy is empty": {
```

Comment:
should we have a test case where we are marshaling the `mfBuild.AdditionalPolicy`